README.md

```diff
@@ -44,6 +44,12 @@ Additional documentation can be found on the [SASjs](https://sasjs.io) website.
   1. `sasjs folder create /Public/folder --target targetName` - creates new new folder.
   2. `sasjs folder delete /Public/folder --target targetName` - deletes folder.
   3. `sasjs folder move /Public/sourceFolder /Public/targetFolder --target targetName` - moves folder to a new location.
+- `sasjs job`: performs job related operations
+  1. `sasjs job execute /Public/job --target targetName` - triggers job for execution.
+     - `sasjs job execute /Public/job --target targetName --wait` - triggers job for execution and waits for job completion.
+     - `sasjs job execute /Public/job --target targetName --output` - triggers job for execution and immediately prints out the response JSON.
+     - `sasjs job execute /Public/job --target targetName --output ./outputFolder` - triggers job for execution and saves the response JSON to a local file.
+     - `sasjs job execute /Public/job --target targetName --log ./logFolder` - riggers job for execution, waits for it to finish, and saves the log to the specified local file.
```

Comment:
missing  a T in "Triggers", also the the example is confusing - does this create a file under `./logFolder` with an unknown name, or does it create a log file called `logFolder` in the current directory?

Similar issue with the line above

Reply:
fixed